### DISCUSSION TITLE
Feature request: Export to CSV option

### DISCUSSION THREAD

**Discussion by vishnus** [postalserver/postal#1465] [Feature suggestions]
Hi team,

I would love to add a feature request, its for exporting the email ids, few meta data as CSV. This helps especially when there are many hardfails / softfails and we'd like to communicate to the clients to check the email client.

**Comment by willpower232** [comment]
I would presume the original intention was for webhooks to report status of messages back to the app sending the messages but it would be good to have more statistics, and exports, available.
